### ISSUE TITLE
Compatibility info appears to be missing, eslint 9 6.x

### ISSUE THREAD

**Issue by replete** [amilajack/eslint-plugin-compat#636]
Is this plugin actually broken?

`eslint.config.js`
```js
...
{
  files: ['dist/*.js'],
  ...compat.configs['flat/recommended'],
  languageOptions: {
    ecmaVersion: 2018,
    globals: {
      ...globals.browser,
  }
  },
  settings: {
    browsers: 'last 10 years'
  }
},
...
```

Run against https://github.com/replete/biscuitman/blob/main/dist/biscuitman.js
yields:
```
/biscuitman/dist/biscuitman.js
  278:16  error  Intl is not supported in Safari 7.1, iOS Safari 8  compat/compat

✖ 1 problems (1 errors, 0 warnings)
```

There are so many incompatible things not being picked up here that something must be wrong. Running this in Chrome 41 (from 2014) yields many javascript errors for instance because it doesn't support arrow functions. 
```
Uncaught SyntaxError: Unexpected token =>
(index):198 Uncaught SyntaxError: Unexpected token )
(index):219 Uncaught SyntaxError: Unexpected token =>
(index):264 Uncaught SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode
```
I've just set 10 years to test if this was working at all, because no errors were coming through. Is this a recent issue or has this been broken for a long time?

**Comment by dkrnl** [comment]
same issues, `settings` config not affect for eslint 9

**Comment by AQS-DTheuke** [comment]
I'm also unable to get any errors from the plugin.

**Comment by GarfieldDeepClone** [comment]
I also have problems getting needed error messages. Reading some threads here I come to the conclusion that this plugin is not working 100% as expected and is therefor useless for me. I have to move on to https://www.npmjs.com/package/eslint-plugin-es-x?activeTab=readme

It's very sad, a plugin that respects the browserslist would have been great.
